static/js/modules/columns.js

```diff
@@ -506,6 +506,26 @@ var reports = {
   }
 };
 
+var loans = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all column--large',
+    render: function (data, type, row) {
+      return row.committee.name;
+    }
+  },
+  {
+    data: 'loan_source_name',
+    orderable: false,
+    className: 'all column--large',
+  },
+  dateColumn({data: 'load_date', orderable: true, className: 'min-tablet hide-panel column--med'}),
```

Comment:
I'm not sure what the value for the `date` column should be, but I don't think `load_date` is right, as that's just about when it was loaded into the system. @PaulClark2, when displaying a table of itemized schedule Cs, should we show a date? And if so, the date of what?